### PR TITLE
Created all blog posts button

### PR DESCRIPTION
**Pull request by NathanCorbin**
issue #33

**Description**
Added a button that will eventually link to all blog posts

**Motivation and Context**
We needed a way to get to all blog posts

**Checklist (check all that is appropriate)**
- [x] Issue is linked in this PR
- [x] Existing tests pass
- [ ] Accessibility tested (if appropriate)

### CODE REVIEW

File: src/components/buttons/__tests__/Button.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import toJson from 'enzyme-to-json'
+import Button from '../Button'
+
+describe('<Button />', () => {
+  it('renders correctly without children', () => {
```
Comment (QuentinGuenther) [inline src/components/buttons/__tests__/Button.spec.tsx:7]: it takes children?

File: src/components/buttons/button-to-blogs/__tests__/ButtonToBlogs.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import toJson from 'enzyme-to-json'
+import ButtonToBlogs from '../ButtonToBlogs'
+
+describe('<ButtonToBlogs />', () => {
+  it('renders correctly without children', () => {
```
Comment (QuentinGuenther) [inline src/components/buttons/button-to-blogs/__tests__/ButtonToBlogs.spec.tsx:7]: This has children?

File: src/components/buttons/button-with-icon/__tests__/ButtonWithIcon.spec.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { ButtonWithIcon } from '../ButtonWithIcon'
 
 describe('<ButtonWithIcon />', () => {
   it('renders correctly without children', () => {
```
Comment (QuentinGuenther) [inline src/components/buttons/button-with-icon/__tests__/ButtonWithIcon.spec.tsx:7]: This has children?

File: src/modules/blog-highlights/excerpt-list/ExcerptList.tsx [hunk 4]
```diff
@@ -33,7 +33,7 @@ const ExcerptList: React.FC<ExcerptListProps> = props => {
             <CardTitle title={title} backgroundUrl={banner} />
             <CardDescription text={description} />
             <CardMeta publishDate={publishDate} />
-            <ButtonWithIcon toUrl="/" />
+            <ButtonWithIcon />
```
Comment (QuentinGuenther) [inline src/modules/blog-highlights/excerpt-list/ExcerptList.tsx:36]: Where did the url go?

File: src/components/buttons/__tests__/Button.spec.tsx (hunk 1 above)
Comment (NathanCorbin) [reply src/components/buttons/__tests__/Button.spec.tsx:7]: copy paste my bad

**Pull request merged**